src/scenedit/scen.main.cpp

```diff
@@ -84,10 +92,90 @@ extern void set_up_apple_events();
 std::unordered_map<std::string, std::shared_ptr <iEventListener>> event_listeners;
 cDrawableManager drawable_mgr;
 
+fs::path scenedit_dir;
+fs::path game_dir;
+fs::path game_binary;
+extern std::string last_load_file;
+
+enum eLaunchType {LOC,START,ENTRANCE};
+
+void launch_scenario(eLaunchType type) {
+	if(boost::ends_with(last_load_file, ".exs")){
+		showError("The scenario editor cannot launch an unpacked scenario directly. You'll need to re-open the scenario from its .boes archive.");
```

Comment:
We should open a new issue for this once the PR is merged, as it seems kind of important to me.